pkg/config/expand_test.go

```diff
@@ -230,7 +230,7 @@ func (s *MySuite) TestApplyUseModules(c *C) {
 
 		// Use ID doesn't exists (fail)
 		g.Modules[len(g.Modules)-1].ID = "wrongID"
-		c.Assert(dc.applyUseModules(), ErrorMatches, fmt.Sprintf("%s: %s", errorMessages["invalidMod"], used.ID))
+		c.Assert(dc.applyUseModules(), ErrorMatches, fmt.Sprintf("%s: %s - Did you mean '%s'\\?", errorMessages["invalidMod"], used.ID, using.ID))
```

Comment:
Let's check.
```
err.Hint == "expected_hint"
errrs.Is(err, InvalidModuleError)
```

Reply:
I think I took care of this in a way that makes sense.  Please review.